whelk-core/src/main/groovy/whelk/converter/marc/RomanizationStep.groovy

```diff
@@ -149,7 +145,7 @@ class RomanizationStep extends MarcFramePostProcStepBase {
         def uniqueTLangs = findUniqueTLangs(thing, byLangPaths)
 
         // TODO: Can multiple transform rules be applied to the same record? What happens then with non-repeatable fields?
-                
+
         uniqueTLangs.each { tLang ->
             unmodifyTLang(thing, tLang, byLangPaths, record)
         }
```

Comment:
I suggest to sort the 880s on their corresponding field number so that they appear in a logical order in the final record. Even though the fields-refs may still be in another order. 
```suggestion
        }
        
        if (thing[HAS_BIB880]) {
            ((List) thing[HAS_BIB880]).sort { bib880 -> bib880[PART_LIST]?[0]?[FIELDREF] ?: '' }
        }
```